dcos_installer/action_lib.py

```diff
@@ -360,8 +360,15 @@ def _add_prereqs_script(chain):
   exit 0
 fi
 
-if ([ x$dist != 'xrhel' ] && [ x$dist != 'xcentos' ]); then
-  echo "$dist is not supported. Only RHEL and CentOS are supported" >&2
+if ([ x$dist != 'xrhel' ] && [ x$dist != 'xcentos' ] && [ x$dist != 'xol' ]); then
+  echo "$dist is not supported. Only RHEL, CentOS and Oracle Enterprise Linux are supported" >&2
```

Comment:
"Supported" implies a lot of things, including testing, bug-fixing, and answering questions. Maybe we could call it "community-supported" instead of "supported" or "Mesosphere-supported", but I'd still expect somebody to be responsible for the above. A softer word would be to say "recommended".
Is @jibsonline running DC/OS on Oracle Enterprise Linux? We could perhaps say "community-tested"?